src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import core.basesyntax.models.Color;
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    private static final int COLOR_COUNT = 8;
+
+    public Color getRandomColor() {
+        return Color.values()[new Random().nextInt(COLOR_COUNT)];
```

Comment:
you instantiate `class Random` every time `getRandomColor` method is invoked.

Should be instantiated once and reused multiple times. (think about making it class-level field)